en/flight_controller/omnibus_f4_sd.md

```diff
@@ -0,0 +1,115 @@
+# Omnibus F4 SD
+The Omnibus F4 SD is a board designed for racers. In contrast to a typical racer
+board it has some additional features, such as an SD card or a faster CPU.
+
+These are the main differences compared to a Pixracer:
+- Lower price
+- Less IO ports (though it's still possible to attach a GPS or a Flow sensor for example)
```

Comment:
Fewer (fewer is used when discussing countable things, while less is used for singular mass nouns.)